src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy

```diff
@@ -720,84 +644,62 @@ class MirrorEndpointsSpec extends HealthCheckSpecification {
         testData << [
                 new MirrorErrorTestData("Unable to create a mirror endpoint on the src sw and sink back to dst sw", {
                     def swPair = switchPairs.all().random()
-                    it.flow = flowHelperV2.randomFlow(swPair)
-                    def freePort = (topology.getAllowedPortsForSwitch(swPair.src) - flow.source.portNumber)[0]
-                    it.mirrorPoint = FlowMirrorPointPayload.builder()
-                            .mirrorPointId(flowHelperV2.generateFlowId())
-                            .mirrorPointDirection(FlowPathDirection.FORWARD.toString())
-                            .mirrorPointSwitchId(swPair.dst.dpId)
-                            .sinkEndpoint(FlowEndpointV2.builder().switchId(swPair.src.dpId).portNumber(freePort)
-                                    .vlanId(flowHelperV2.randomVlan())
-                                    .build())
-                            .build()
+                    def flowEntity = flowFactory.getBuilder(swPair).build()
+                    it.flow = flowEntity
```

Comment:
it.flow = flowFactory.getBuilder(swPair).build()

Reply:
+